vsphere/resource_vsphere_compute_cluster_test.go

```diff
@@ -304,6 +304,28 @@ func TestAccResourceVSphereComputeCluster_vsanDITEncryption(t *testing.T) {
 	})
 }
 
+func TestAccResourceVSphereComputeCluster_vsanEsaEnabled(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
```

Comment:
We can perform a pre-check to skip the test if vSphere is not >= 8.0. Or we could just have the test error in our CI (which wouldn't be the end of the world). Once CI is running against 8.0 it will pass/be worth looking into when it fails.

Reply:
> We can perform a pre-check to skip the test if vSphere is not >= 8.0. Or we could just have the test error in our CI (which wouldn't be the end of the world). Once CI is running against 8.0 it will pass/be worth looking into when it fails.

If we could ignore its result, I prefer to keep it here.

Reply:
Resolved with https://github.com/hashicorp/terraform-provider-vsphere/pull/1874/commits/f6adc75c6454cb760f2ecd27072c54a4fa37aecb